File: 8kyu/do-i-get-a-bonus/do-i-get-a-bonus.go
package main

import (
	"fmt"
	"strconv"
)

// func BonusTime(salary int, bonus bool) string {
//   var total string
// 	if bonus {
// 		total = strconv.Itoa(salary * 10)
// 	} else {
// 		total = strconv.Itoa(salary)
// 	}

// 	return "£" + total
// }

func BonusTime(salary int, bonus bool) string {
  total := strconv.Itoa(salary)

	if bonus {
		total = strconv.Itoa(salary * 10)
	}
	
	return "£" + total
}

func main() {
	fmt.Println(BonusTime(2, true))
	fmt.Println(BonusTime(78, false))
}

File: 8kyu/do-i-get-a-bonus/do-i-get-a-bonus.py
# def bonus_time(salary: int, bonus: int) -> str:
#   total = salary * 10 if bonus else salary
#   return f"${total}"

def bonus_time(salary: int, bonus: int) -> str:
  return "${}".format(salary * (10 if bonus else 1))

print(bonus_time(2, True))
print(bonus_time(78, False))